example/src/test/scala/monocle/function/AtExample.scala

```diff
@@ -34,4 +38,21 @@ class AtExample extends MonocleSuite {
     (Set(1, 2, 3) applyLens at(2) set None)     shouldEqual Set(1, 3)
   }
 
+  test("at creates a Lens from Int to one of its bit") {
+    (3 applyLens at(0: IntBits) get) shouldEqual true   // true  means bit is 1
+    (4 applyLens at(0: IntBits) get) shouldEqual false  // false means bit is 0
+
+
+    (32 applyLens at(0: IntBits) set true)   shouldEqual 33
+    (3  applyLens at(1: IntBits) modify(!_)) shouldEqual 1 // toggle 2nd bit
+
+    illTyped("""0 applyLens at(79) get""")
```

Comment:
One underused feature of `illTyped` (even in shapeless) is that it takes as optional second argument a regular expression that is matched against the compiler error. This guards against syntax errors in the code which also let `illTyped` succeed. [Here](https://github.com/fthomas/refined/blob/28d1e3802b0a1d00e3b848668e35993e88d4fc5f/core/shared/src/test/scala/eu/timepit/refined/RefineMSpec.scala#L71) is an example.